src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Xamarin.Android.Sdk.props

```diff
@@ -22,8 +22,9 @@ Copyright (C) 2011-2016 Xamarin. All rights reserved.
 	<PropertyGroup>
 		<!-- Until VS2017+ includes its own ReferenceAssemblies outside of C:\Program Files (x86)\Reference Assemblies and into 
 			 the VsInstallRoot, we must override this ourselves for our SDKs -->
-		<TargetFrameworkRootPath Condition="'$(VsInstallRoot)' != '' And '$(TargetFrameworkRootPath)' == ''">$(VsInstallRoot)\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\</TargetFrameworkRootPath>
-		<TargetFrameworkRootPath Condition="'$(VsInstallRoot)' == '' And '$(TargetFrameworkRootPath)' == ''">$(ProgramFiles)\Reference Assemblies\Microsoft\Framework\</TargetFrameworkRootPath>
+		<TargetFrameworkRootPath Condition="'$(HostOS)' == 'Windows' And '$(VsInstallRoot)' != '' And '$(TargetFrameworkRootPath)' == ''">$(VsInstallRoot)\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\</TargetFrameworkRootPath>
```

Comment:
This change doesn't make sense to me, as it appears to be intermixing "build" environment with "installed" environment.

`$(HostOS)` is only defined by the xamarin-android build system. Once installed (from `.pkg`, `.vsix`, `make install`, etc.), `$(HostOS)` *will not be defined*, and thus this will be the empty string.

As such, this patch will likely break Visual Studio/Windows.

Using `'$(OS)' == 'Windows_NT'` instead will probably work.

Reply:
Ok, just made it so.